server/middleware/setUpSentry.ts

```diff
@@ -14,7 +14,7 @@ function setUpSentryRequestHandler(app: express.Express): void {
   // Prevent usernames which are PII from being sent to Sentry
```

Comment:
Is this from the thing Rich shared? Is it worth linking to the Sentry guidance in the commit message or a comment?

Also, the commit message says "Sentry is still being usernames" 👀